### PR TITLE
meta-oe: fbida: switch to git and meson, update to newer SRC_REV

### PR DESCRIPTION
**Pull request by rohieb**
fbida has not seen any versioned releases for five years even though development has not stalled. Update to a recent version, switch to meson build, and make fbpdf build optional.

### CODE REVIEW

File: meta-oe/recipes-graphics/fbida/files/0002-meson.build-add-features-options-for-png-gif-tiff-we.patch [hunk 1]
```diff
@@ -0,0 +1,123 @@
+From fa60c06cadd6acd8abede91e75254e7250a087a3 Mon Sep 17 00:00:00 2001
+From: "Meyer, Wolfgang" <Wolfgang.Meyer@gossenmetrawatt.com>
+Date: Fri, 16 Sep 2022 13:40:25 +0200
+Subject: [PATCH] meson.build: add features options for png, gif, tiff, webp,
+ and motif
+
```
Comment (kraj) [inline meta-oe/recipes-graphics/fbida/files/0002-meson.build-add-features-options-for-png-gif-tiff-we.patch:6]: follow https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Header_Recommendations:_Upstream-Status

**Review by kraj — CHANGES_REQUESTED** [review]

**Comment by kraj** [comment]
it fails to build for musl targets see [1]

```
../git/gfx.h:43:5: error: unknown type name 'dev_t'; did you mean 'div_t'?
    dev_t devnum;
    ^~~~~
    div_t
TOPDIR/build/tmp/work/cortexa57-yoe-linux-musl/fbida/2.14+gitAUTOINC+eb769e3d7f-r0/recipe-sysroot/usr/include/stdlib.h:64:35: note: 'div_t' declared here
typedef struct { int quot, rem; } div_t;
                                  ^
1 error generated
```

I think we need to add `#include <sys/types.h>` in `gfx.h` to fix it.

[1] https://errors.yoctoproject.org/Errors/Details/699643/

**Comment by kraj** [comment]
I have addressed the comments as needed and accepted this change at 28b86939d5 on master branch

**Comment by rohieb** [comment]
Thank you, sorry it fell through the cracks on my side…

**Pull request closed without merging**